packages/enzyme-test-suite/package.json

```diff
@@ -49,4 +49,4 @@
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-react": "^7.5.1"
   }
-}
+}
```

Comment:
let's revert this file; every file should always have a trailing newline

Reply:
The diff is added again😉 I'm not digging into that but I guess it is added by a script to run tests.

Reply:
ah, weird. maybe npm itself. not a big deal.